src/raw/config-avalanche.js

```diff
@@ -0,0 +1,201 @@
+      description: "Allows the burning of FLT tokens for a proportional share of the value held in the treasury.",
+      codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/Treasury_v0.sol",
+    },
+    gems: {
+      address: "0xB779a4A28012e12Ff904754bBE72F60423AF0A28",
+      description: "Contract giving user GEMS, inspired by https://github.com/andrecronje/rarity/blob/main/rarity.sol",
+      codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/GEMS.sol",
+    },
+    gemCollectorNft: {
+      address: "0xEB73D5CbB26536F714785B7864806542f41ab9E5",
+      description: "NFTs for users who have accrued enough GEMS, a new NFT for each level of GEM accrual.",
+      codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/GemCollectorNFT.sol",
+    },
+  },
+  paymentTokens: [{
+    address: "0xd586e7f844cea2f87f50152665bcbc2c279d8d70",
+    isUSDBased: true,
+    decimals: 18,
+    symbol: "DAI",
+    iconUrl: "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/icons/tokens/dai.svg",
+  }],
+  hiddenMarkets: [],
+  markets: [
+    {
+      index: 1,
+      name: "Avalanche Market",
+      description:
+        "Avax is the main token of the Avalanche blockchain and acts as the primary 'fuel' that powers all activity on it. Avax is used to perform actions on the network, such as sending transactions and interacting with dapps.",
+      iconUrl: "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/tokens/avax/avax.png",
+      leverage: 2,
+      paymentTokenSymbol: "DAI",
+      symbol: "2AVAX",
+      launchTimestamp: 1639490400,
+      createdTimestamp: 1638805641,
+      yieldManager: {
+        address: "0x621cda10820555adAe8506eeC296cB9621E491Ff",
+        description: "Aave-based yield manager",
+        codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/YieldManagers/YieldManagerAaveBasic.sol",
+      },
+      oracle: {
+        decimals: 8,
+        heartbeat: 135,
+      },
+      longToken: {
+        address: "0xa43a73f17ECde987A9127A5cBE46B7923F351c32",
+        symbol: "flAvax2",
+        iconUrl:
+          "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/icons/tokens/avax/avax_long.png",
+      },
+      shortToken: {
+        address: "0xdC883B026F78FF494199778001b23f38edA89d6d",
+        symbol: "fsAvax2",
+        iconUrl:
+          "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/icons/tokens/avax/avax_short.png",
+      },
+    },
+    {
+      index: 2,
+      name: "JOE Market",
+      description:
+        "JOE is a governance token that rewards its holders with a share of exchange revenues from the Trader Joe protocol. Trader Joe is a one-stop decentralized trading platform on the Avalanche network.",
+      oracleHeartbeatInSec: 1800,
+      iconUrl: "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/tokens/joe/joe.png",
+      oracleDecimals: 8,
```

Comment:
Is this maybe a mistake? Should this not be inside an oracle type?

Reply:
Yes it is a mistake, will remove